Mage/src/main/java/mage/abilities/effects/ContinuousEffectImpl.java

```diff
@@ -516,9 +518,18 @@ public boolean canLookAtNextTopLibraryCard(Game game) {
         return true;
     }
 
+    public boolean hasHint() {
+        return false;
```

Comment:
So what's the rationale for this method, versus just checking if `getAffectedHints()` is empty?

Reply:
I did it that way initially, but Jaydi asked to change it to this.